toolflow/vivado/common/design.master.tcl.template

```diff
@@ -111,6 +111,11 @@ if {[llength [info commands platform::generate_wrapper]] == 0} {
   platform::generate_wrapper
 }
 
+# skip platform generation if flag was provided
+if {@@SKIP_SYNTH@@ > 0} {
+	exit
+}
+
```

Comment:
While this does exactly what it is supposed to do (ie. allow skipping of the synthesis) it is still very "unclean". If possible, define a corresponding variable at the start of the script (`set skip_synth true/false`, roughly where tapasco_freq/host_freq are defined) with a corresponding comment. This would allow the user to easily change the flag manually to run a synthesis within Vivado. Additionally, it would be a lot cleaner.